### PR TITLE
#183 add node-rpc-method ClaimFromRewardingFund

### LINKED ISSUE #183: add node-rpc-method ClaimFromRewardingFund

### PR DESCRIPTION
**Pull request by uu-z**
**What type of PR is this?**

> Uncomment only one `/kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #183 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by uu-z** [comment]
iotexproject/web-bp#824

File: src/__test__/iotx.integration.test.ts [hunk 1]
```diff
@@ -179,3 +179,18 @@ accountTest("readContractByMethod", async t => {
...
+  const antenna = new Antenna(IOTEX_CORE);
+  const sender = antenna.iotx.accounts.privateKeyToAccount(
+    TEST_PRIVATE_KEY_HAVING_IOTX
+  );
+  const hash = await antenna.iotx.ClaimFromRewardingFund({
+    from: sender.address,
+    amount: "0",
+    gasPrice: "1",
+    gasLimit: "1000000",
+    data: Buffer.from("test", "hex")
```
Comment (puncsky) [inline src/__test__/iotx.integration.test.ts]: "test" is actually not hex

**Comment by ququzone** [comment]
miss action sign process, should test your local before push.

**Comment by uu-z** [comment]
> miss action sign process, should test your local before push.

it's tested in local before push and  i saw the sign process is already includes in sendAction.

**Pull request merged**